protocol/0080-SPOT-product_builtin_spot.md

```diff
@@ -63,61 +64,23 @@ submission = {
 
 ### Amendments and Cancellations
 
 A liquidity amendment or a cancellation is determined as valid following spec [0044-LIME](./0044-LIME-lp_mechanics.md) with the following exceptions:
 
-- the `maximum_reduction_amount` should be expressed in the `quote_asset` when reducing the `buy_commitment_amount` and the `base_asset` when reducing the `sell_commitment_amount`.
-- the `maximum_reduction_amount` should be `INF` in the case where the liquidity `time_window=Os` (Note: this is not strictly necessary but an LP is effectively able to reduce their commitment to zero anyway in this case through multiple commitments.)
-
-```pseudo
-Market Data:
+- the `maximum_reduction_amount` should be `INF` in the case where the liquidity `time_window=Os` (Note: this is not strictly necessary but an LP is effectively able to reduce their commitment to zero anyway in this case through multiple commitment amendments)
 
-    base_asset: ETH
-    quote_asset: DAI
-
-    mark_price = 1000
-
-Market Liquidity:
-
-    total_stake = 100,000 DAI (or 100 ETH)
-    target_stake = 25,000 DAI (or 25 ETH)
-
-    maximum_reduction_amount = 1000,000 - 25,0000 = 75,000 DAI (or 75 ETH)
-```
-
-`virtual_stake` values should be updated during liquidity amendments or cancellations following the mechanisms detailed in spec [0044-LIME](./0044-LIME-lp_mechanics.md) with the following exceptions:
-
-- reducing the `buy_commitment_amount` only reduces the `buy_virtual_stake`
-- reducing the `sell_commitment_amount` only reduces the `sell_virtual_stake`
-
-A single LP will never be able to reduce their commitment to `0`. They can either keep reducing to a sufficiently small amount they're willing to ignore, or they can submit a governance vote to cancel the market, see the [governance spec](./0028-GOVE-governance.md).
+As the target stake (and therefore `maximum_reduction_amount`) is some factor of the total stake (see [Target Stake](./0080-SPOT-product_builtin_spot.md#market-target-stake)), a single LP will never be able to reduce their commitment to `0` if they are the only LP in a market. They can either keep reducing to a sufficiently small amount they're willing to ignore, or they can submit a governance vote to cancel the market, see the [governance spec](./0028-GOVE-governance.md).
 
 For market cancellation proposal a sole LP in the market holds all the voting power (unless governance token holders override them).
 
-### Liquidity Shortfalls
-
-If at any point in time, a liquidity provider has insufficient capital in their general accounts to cover a transfer arising from a filled liquidity order, the network will utilise the liquidity commitment, held in the relevant bond account to cover the shortfall, applying the bond penalty factor (slashing the bond).
-
-As there is no market insurance pool, funds from bond slashing in the result of shortfall will be transferred to the global network treasury for that asset.
-
```

Comment:
This is the biggest implication of going with the single commitment amount route as a result of the SLA changes @davidsiska-vega @barnabee 

An LPs bond account can no longer be used to cover any shortfalls an LP may have when submitting orders (they need to transfer the full value of the order to a holding account).

Basically means an LP needs at least 3x their commitment amount to support their commitment.

Reply:
And that's fine, it just means they're not meeting their SLA. They will be penalised as per the spec.